src/components/Quantity/Quantity.jsx

```diff
@@ -1,11 +1,26 @@
 import { useState } from "react";
+import React from "react";
 import Input from "../Input/Input";
 import Button from "../Button/Button";
 
 import "./Quantity.scss";
 
+export const AuthContext = React.createContext({});
+
+export const AuthPorvider = (props) => {
+  const cart = {
+    quantity: 1,
+  };

```

Comment:
```suggestion
  const [cart, setCart] = useState(cartDefaultValue) // import this from context file. import {cartDefaultValue} from ''
```

We create a state here because we want to change it when a new product is added to the cart, so later we'll use it to update